docs/paypal.md

```diff
@@ -0,0 +1,173 @@
+# Pay with PayPal
+
+Accept PayPal payments in your iOS app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
```

Comment:
This might be a better link for `countries and currencies`:  https://www.paypal.com/us/webapps/mpp/country-worldwide

The current link is kind of confusing. It doesn't seem to show the countries that PayPal is available in, but rather feature support in different countries